ipaserver/install/ipa_replica_prepare.py

```diff
@@ -601,7 +601,8 @@ def export_certdb(self, fname, passwd_fname, is_kdc=False):
                 api.env.realm, nssdir=self.dir, subject_base=subject_base)
             db.create_passwd_file()
             ca_db = certs.CertDB(
-                api.env.realm, host_name=api.env.host,
+                api.env.realm, paths.HTTPD_ALIAS_DIR,
+                host_name=api.env.host,
                 subject_base=subject_base)
             db.create_from_cacert()
             db.create_server_cert(nickname, hostname, ca_db)
```

Comment:
Rather than using `/etc/httpd/alias` as `ca_db` here, update `CertDB.create_server_cert()` not to require the `other_certdb` argument.